### PR TITLE
TODO items annotated with referencing issue number

### PR DESCRIPTION
**Pull request by rajarshimaitra**

### CODE REVIEW

**Comment by rajarshimaitra** [comment]
Updated after removing Issue duplication and modified annotation.

File: src/fungibled/runtime.rs [hunk 1]
```diff
@@ -528,9 +529,10 @@ impl Runtime {
         let reply = self
             .stash_req_rep(rpc::stash::Request::Enclose(disclosure.clone()))?;
         if let Reply::Success = reply {
-            // TODO: Improve RGB Core disclosure API providing methods for
+            // TODO #156: Improve RGB Core disclosure API providing methods for
             // indexing underlying       data in different ways. Do
             // the same for Consignment
```
Comment (dr-orlovsky) [inline src/fungibled/runtime.rs:534]:
```suggestion
            // TODO #156: Improve RGB Core disclosure API providing methods for
            //      indexing underlying data in different ways. Do
            //      the same for Consignment
```

**Review by dr-orlovsky — CHANGES_REQUESTED** [review]
One of TODOs was broken by `fmt` thing

**Comment by rajarshimaitra** [comment]
Updated..

File: src/error.rs [hunk 2]
```diff
@@ -11,7 +11,8 @@
 // along with this software.
 // If not, see <https://opensource.org/licenses/MIT>.
 
-// TODO: Consider moving parts of this file to common daemon modules (LNP/BP)
+// TODO #152: Consider moving parts of this file to common daemon modules
+// (LNP/BP)
```
Comment (dr-orlovsky) [inline src/error.rs:15]:
```suggestion
// TODO #152: Consider moving parts of this file to common daemon modules
//       (Internet2)
```

File: src/fungibled/runtime.rs [hunk 3]
```diff
@@ -234,8 +234,8 @@ impl Runtime {
     ) -> Result<Reply, ServiceErrorDomain> {
         debug!("Got TRANSFER {}", transfer);
 
-        // TODO: Check inputs that they really exist and have sufficient amount
-        //       of asset for the transfer operation
+        // TODO #66: Check inputs that they really exist and have sufficient
+        // amount of asset for the transfer operation
```
Comment (dr-orlovsky) [inline src/fungibled/runtime.rs:238]:
```suggestion
        // TODO #66: Check inputs that they really exist and have sufficient
        //       amount of asset for the transfer operation
```

File: src/fungibled/runtime.rs [hunk 4]
```diff
@@ -528,9 +528,9 @@ impl Runtime {
         let reply = self
             .stash_req_rep(rpc::stash::Request::Enclose(disclosure.clone()))?;
         if let Reply::Success = reply {
-            // TODO: Improve RGB Core disclosure API providing methods for
-            // indexing underlying       data in different ways. Do
-            // the same for Consignment
+            // TODO #156: Improve RGB Core disclosure API providing methods for
+            // indexing underlying data in different ways. Do the same for
+            // Consignment
```
Comment (dr-orlovsky) [inline src/fungibled/runtime.rs:533]:
```suggestion
            // TODO #156: Improve RGB Core disclosure API providing methods for
            //       indexing underlying data in different ways. Do the same for
            //       Consignment
```

File: src/i9n/runtime.rs [hunk 5]
```diff
@@ -39,8 +39,8 @@ impl Runtime {
                 data_dir: config.data_dir.clone(),
                 verbose: config.verbose,
                 electrum_server: config.electrum_server.clone(),
-                // TODO: Endpoint parameters are not needed in embedded mode;
-                //       remove them
+                // TODO #159: Endpoint parameters are not needed in embedded
+                // mode; remove them
```
Comment (dr-orlovsky) [inline src/i9n/runtime.rs:43]:
```suggestion
                // TODO #159: Endpoint parameters are not needed in embedded
                //       mode; remove them
```

File: src/rpc/reply.rs [hunk 6]
```diff
@@ -136,8 +136,8 @@ pub struct Failure {
 
 impl From<internet2::presentation::Error> for Failure {
     fn from(err: internet2::presentation::Error) -> Self {
-        // TODO: Save error code taken from `Error::to_value()` after
-        //       implementation of `ToValue` trait and derive macro for enums
+        // TODO #61: Save error code taken from `Error::to_value()` after
+        // implementation of `ToValue` trait and derive macro for enums
```
Comment (dr-orlovsky) [inline src/rpc/reply.rs:140]:
```suggestion
        // TODO #61: Save error code taken from `Error::to_value()` after
        //       implementation of `ToValue` trait and derive macro for enums
```

File: src/rpc/reply.rs [hunk 7]
```diff
@@ -147,8 +147,8 @@ impl From<internet2::presentation::Error> for Failure {
 
 impl From<internet2::transport::Error> for Failure {
     fn from(err: internet2::transport::Error) -> Self {
-        // TODO: Save error code taken from `Error::to_value()` after
-        //       implementation of `ToValue` trait and derive macro for enums
+        // TODO #61: Save error code taken from `Error::to_value()` after
+        // implementation of `ToValue` trait and derive macro for enums
```
Comment (dr-orlovsky) [inline src/rpc/reply.rs:151]:
```suggestion
        // TODO #61: Save error code taken from `Error::to_value()` after
        //       implementation of `ToValue` trait and derive macro for enums
```

File: src/rpc/reply.rs [hunk 8]
```diff
@@ -159,8 +159,8 @@ impl From<internet2::transport::Error> for Failure {
 #[cfg(feature = "node")]
 impl From<RuntimeError> for Failure {
     fn from(err: RuntimeError) -> Self {
-        // TODO: Save error code taken from `Error::to_value()` after
-        //       implementation of `ToValue` trait and derive macro for enums
+        // TODO #61: Save error code taken from `Error::to_value()` after
+        // implementation of `ToValue` trait and derive macro for enums
```
Comment (dr-orlovsky) [inline src/rpc/reply.rs:163]:
```suggestion
        // TODO #61: Save error code taken from `Error::to_value()` after
        //       implementation of `ToValue` trait and derive macro for enums
```

File: src/rpc/reply.rs [hunk 9]
```diff
@@ -171,8 +171,8 @@ impl From<RuntimeError> for Failure {
 #[cfg(any(feature = "node", feature = "client"))]
 impl From<ServiceError> for Failure {
     fn from(err: ServiceError) -> Self {
-        // TODO: Save error code taken from `Error::to_value()` after
-        //       implementation of `ToValue` trait and derive macro for enums
+        // TODO #61: Save error code taken from `Error::to_value()` after
+        // implementation of `ToValue` trait and derive macro for enums
```
Comment (dr-orlovsky) [inline src/rpc/reply.rs:175]:
```suggestion
        // TODO #61: Save error code taken from `Error::to_value()` after
        //       implementation of `ToValue` trait and derive macro for enums
```

File: src/stashd/index/btree.rs [hunk 10]
```diff
@@ -45,7 +45,8 @@ use crate::util::file::{file, FileMode};
 )]
 #[strict_encoding_crate(lnpbp::strict_encoding)]
 struct BTreeIndexData {
-    /// TODO: Replace with DisplayFromStr once RGB node will fix node display
+    /// TODO #164: Replace with DisplayFromStr once RGB node will fix node
+    /// display
```
Comment (dr-orlovsky) [inline src/stashd/index/btree.rs:49]:
```suggestion
    /// TODO #164: Replace with DisplayFromStr once RGB node will fix node
    ///       display
```

File: src/stashd/runtime.rs [hunk 11]
```diff
@@ -338,10 +338,10 @@ impl Runtime {
...
         debug!("Got FORGET");
 
-        // TODO: Implement stash prunning: filter all transitions containing
-        //       revealed outpoints from the removal_list, and if they do
-        //       not have any other _known_ outpoints, remove them — and iterate
-        //       over their direct ancestor in the same manner
+        // TODO #63: Implement stash prunning: filter all transitions containing
+        // revealed outpoints from the removal_list, and if they do
+        // not have any other _known_ outpoints, remove them — and iterate
+        // over their direct ancestor in the same manner
```
Comment (dr-orlovsky) [inline src/stashd/runtime.rs:344]:
```suggestion
        // TODO #63: Implement stash prunning: filter all transitions containing
        //       revealed outpoints from the removal_list, and if they do
        //       not have any other _known_ outpoints, remove them — and iterate
        //       over their direct ancestor in the same manner
```

**Review by dr-orlovsky — CHANGES_REQUESTED** [review]

**Review by dr-orlovsky — APPROVED** [review]

**Review by dr-orlovsky — APPROVED** [review]

**Pull request merged**